src/components/BounceDetailsContainer/index.jsx

```diff
@@ -47,14 +47,29 @@ const BounceRuleDetailed = ({
   updatePageIndex,
   filteredChangelog,
   logout,
+  isNetworkError,
+  handleRevertModalClose,
   handleDropdownSelect,
   isCommitValid,
-  onEditRuleCommit,
 }) => {
   const { id } = currentRule;
   const isChangelogEmpty = changelog === undefined || changelog.length < 1;
   return (
     <div>
+      {isNetworkError && (
+        <div className="network-alert">
```

Comment:
since this alert banner is being reused, we can make it its own component and then pass in props for things that may be different, like the `href`